src/App.vue

```diff
@@ -22,6 +22,7 @@
 
       <!-- SBC Common Components header -->
       <SbcHeader
+        id="namerequest-sbc-header"
         class="sbc-header"
```

Comment:
Could you clean this up please?
- remove class
- change selector in local (scoped) style block from class to id
- move sbc-header styles from overrides.scss to local style block

Reply:
Or... add a common mixin method to "scrollByClassname".

Reply:
Meh, "scroll by id" is better since ids should be unique.

Another idea: scroll to "app" id instead of the sbc header???

Reply:
yeah sure

Reply:
I did try scrolling to "app" but unfortunately it would go to the centre of it which is below the top page since it encompasses everything